app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -54,12 +55,10 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener {
         sessionManager.sessions.observe(this, object : NonNullObserver<List<Session>>() {
             public override fun onValueChanged(sessions: List<Session>) {
                 if (sessions.isEmpty()) {
-                    // There's no active session. Show the URL input screen so that the user can
-                    // start a new session.
-                    ScreenController.showHomeScreen(supportFragmentManager, this@MainActivity)
-                } else {
-                    ScreenController.showBrowserScreenForCurrentSession(supportFragmentManager, sessionManager)
+                    // There's no active session. Start a new session with "homepage".
+                    sessionManager.createSession(Source.USER_ENTERED, URL_HOME)
                 }
+                ScreenController.showBrowserScreenForCurrentSession(supportFragmentManager, sessionManager)
```

Comment:
nit: This is going to get called twice: once now and once when the session is created. I think you should do this instead:
```kotlin
if (sessions.isEmpty()) {
    ScreenController.showBrowserScreenForUrl(URL_HOME)
} else {
    ScreenController.showBrowserScreenForCurrentSession(...)
}